CoreScriptsRoot/Modules/Settings/Pages/Players.lua

```diff
@@ -274,7 +274,7 @@ local function Initialize()
 					table.insert(existingPlayerLabels, index, frame)
 				end
 				frame.Name = 'PlayerLabel'..player.Name
-				frame.Icon.Image = 'http://www.roblox.com/Thumbs/Avatar.ashx?x=100&y=100&userId='..math.max(1, player.userId)
+				frame.Icon.Image = 'http://www.roblox.com/Thumbs/Avatar.ashx?x=100&y=100&userId='..tostring(player.userId)
```

Comment:
The userId is still a valid number. Not sure how the API endpoint reacts to getting a -1 though.